lib/parser.js

```diff
@@ -9,7 +9,7 @@ exports.initialtNumber = initialtNumber;
 exports.createNumber = createNumber;
 exports.TextRange = TextRange;
 
-function parseText(text) {
+function parseText(text, referenceCaps, verseCaps) {
```

Comment:
Would prefer
```suggestion
function parseText(text, options = {}) {
```

so the options are all in the `options` object. Then use the function
```
parseText(text, { referenceCaps: false, verseCaps: true })
```